docs/src/ITensorType.md

```diff
@@ -26,7 +26,7 @@ itensor(::Array{<:Number}, ::ITensors.Indices)
 ## QN BlockSparse Constructors
 
 ```@docs
-ITensor(::Type{<:Number}, ::QN, ::ITensors.Indices)
+ITensor(::Type{<:Number}, ::QN, ::ITensors.QNIndices)
```

Comment:
The doc string in the source is associated with this version of the function. Without this change there is an error because the original version has no doc string.

Reply:
Sounds good, thanks.